examples/creatives/nonlinear_overlay.js

```diff
@@ -6,4 +6,20 @@ class NonLinear extends BaseSimidCreative {
     constructor() {
       super();
     }
+
+    onRequestResize() {
+      const resize_x_val = document.getElementById('resize_x_val').value;
+      const resize_y_val = document.getElementById('resize_y_val').value;
+      const resize_width = document.getElementById('resize_width').value;
+      const resize_height = document.getElementById('resize_height').value;
+  
+      const resize_params = {
+        'x_val': resize_x_val,
```

Comment:
https://interactiveadvertisingbureau.github.io/SIMID/#simid-creative-requestResize

The keys should be x and y, not x_val and y_val

Reply:
fixed